spec/classes/config_spec.rb

```diff
@@ -20,12 +20,72 @@
         )
     }
 
-    it {
-      is_expected.to contain_exec('wsus-config-update-languages')
-        .with(
-          logoutput: true,
-          provider: 'powershell',
-        )
-    }
+    it { should contain_exec('wsus-config-update-synchronization').with({
```

Comment:
Lets remove the checking of the command and unless attribute content from the test.  This stuff is super hard to maintain and gives way more headache and little value.